guides/micronaut-data-hibernate-reactive/java/src/main/java/example/micronaut/domain/Genre.java

```diff
@@ -8,6 +9,7 @@
 import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
 
+@Serdeable
```

Comment:
I cannot see why we now need to add `@Serdeable` here, but previously it worked without `@Introspected`